python/ray/experimental/state.py

```diff
@@ -751,22 +761,13 @@ def cluster_resources(self):
                 resource in the cluster.
         """
         resources = defaultdict(int)
-        clients = self.client_table()
-        for client in clients:
-            # Only count resources from live clients.
-            if client["IsInsertion"]:
-                for key, value in client["Resources"].items():
-                    resources[key] += value
+        live_clients, _ = self.client_table()
+        for client in live_clients:
+            for key, value in client["Resources"].items():
+                resources[key] += value
 
         return dict(resources)
 
-    def _live_client_ids(self):
```

Comment:
I think you should add a public function for `live_client_info()` that just returns the first item of the `client_table()`. It's a very common pattern, even in this PR, and makes the code much more readable.